web/src/beta/features/Editor/tabs/publish/Nav/PublishModal/index.tsx

```diff
@@ -76,15 +73,16 @@ const PublishModal: React.FC<Props> = ({
     [purl],
   );
 
-  const publishDisabled = useMemo(
-    () =>
-      loading ||
-      (publishing === "unpublishing" && publishStatus === "unpublished") ||
-      ((publishing === "publishing" || publishing === "updating") &&
-        (!alias || !!validation || validatingAlias || !validAlias)),
-    [alias, loading, publishStatus, publishing, validation, validAlias, validatingAlias],
-  );
+  // const publishDisabled = useMemo(
+  //   () =>
```

Comment:
@KaWaite I commented this code cause when you unpublish the project the button remain disabled until you reload the page I though it best only to use loading

Reply:
Without this the user can send the publish mutation even if the alias isn't valid, etc which isn't good.
Can you figure out what is either what conditional in this useMemo is incorrect and/or what value isn't being updated correctly? 🙏🏻

Reply:
@KaWaite I fixed it this was the line(  (publishing === "unpublishing" && publishStatus === "unpublished") )
 that was causing to condition to remain true after unpublish.